master-config.yaml-sample

```diff
@@ -10,7 +10,7 @@ builders:
 - aarch64-rhel-9
 - aarch64-ubuntu-2004
 - aarch64-ubuntu-2204
-- aarch64-ubuntu-2304
+- aarch64-ubuntu-2310
```

Comment:
is 2310 already set up?

Reply:
triggered it here https://buildbot.dev.mariadb.org/#/builders/2/builds/5

Is that all the setup required?

Reply:
master-config.yaml-sample is just a sample file that only allows running `buildbot checkconfig` for the root master directory for the master.cfg template file that is used to generated all masters from `autogen`. So, the builder should run without any changes to this file. Maybe it would make sense to even remove it